### PR TITLE
adding eslint

### PR DESCRIPTION
**Pull request by dbrudner**
I added eslint to the project. Here's a couple notes:

- Extends airbnb rules
- Set to use 2 space for indentation, per the pattern established in the project.
- Set to use single quotes. There wasn't consistency across the project, I selected single quotes. It's easy to revert to double quotes by changing the setting in .eslintrc.json and running eslint --fix.
- I changed files using jsx to have the extension .jsx instead of .js
- I ran eslint --fix and committed the changes made by that command.
- Instead of fixing the errors came up, I elected to ignore the rules inside .eslintrc.json. I did so because firstly, I don't know how the contributors wish to define these rules, and secondly, that potentially falls outside of the scope of the issue. There's some decisions that'll have to be made there, and I don't think I should be the person to make them.

I'd recommend moving forward that whoever's project this is make decisions about the linting rules, then remove/change the rules set in .eslintrc.json and fix the files so the code complies with the rules decided upon.

Having said that, thank you for letting me contribute. If you have any questions, comments, please let me know or comment directly in the PR and I can make changes before merging.

### CODE REVIEW

**Comment by gregorysl** [comment]
I'm curious, why 

> I changed files using jsx to have the extension .jsx instead of .js

**Comment by dbrudner** [comment]
@gregorysl It was caught by the linter and easy to change, so I made the change. JavaScript files containing jsx should contain that file extension. Feel free to revert if you'd like.

**Comment by dbrudner** [comment]
Thanks! Definitely keep the file names as .js if that's what you prefer! There's definitely a good argument for doing so!

Much love, good luck in the future. I'll check for other issues when you post them and continue contributing if there's anything that interests me!

File: src/features/Issues/Issues.jsx [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { getIssues, filterIssues } from './actions';
+
+const maxLength = 50;
+
+const Card = ({
+  title, body, labels, issueUrl, owner, repo, number,
+}) => (
+=======
```
Comment (dillionverma) [inline src/features/Issues/Issues.jsx]: Some merge conflicts remaining

**Comment by dbrudner** [comment]
Resolved the conflicts, looks good to go from my end! Comment away if there's something I missed. Other than that, I completely support the decision to keep using .js files and think there's definitely a good argument for doing so. I enjoyed contributing here and would be interested in tackling more issues in the future!

**Comment by dbrudner** [comment]
Encountered a second error after fixing this, should be good to go now!

**Comment by dillionverma** [comment]
Merged it in, thanks again so much for your help!

**Pull request merged**